tests/testthat/test-utils_tfrmt.R

```diff
@@ -186,7 +189,8 @@ test_that("Check apply_tfrmt for mock data",{
     )
   )
   mock_dat <- make_mock_data(plan, .default = 1, n_col = 1) %>%
-    apply_tfrmt(plan, mock =TRUE)
+    quietly(apply_tfrmt)(plan, mock =TRUE) %>%
```

Comment:
Since you used quietly() was this supposed to return a message?

Reply:
it is returning the message about the ways to correct multiple params per group/label/col. That isnt the point of the test, so I wanted it to be quiet. I find that unsuppressed non-relevant messages clutter up the ability to read and understand the testthat output

Reply:
yep that makes sense! i was just running it locally and there was no message captured by the quietly.